eva_submission/eload_preparation.py

```diff
@@ -3,7 +3,6 @@
 import shutil
 
 from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_ensembl
-
```

Comment:
This is _very_ unimportant but you might as well leave this empty line in here, as it saves you from having to touch this file at all and keeps it consistent with the other files (separating internal imports from external libraries).

Reply:
Okay